src/components/Calculator.js

```diff
@@ -0,0 +1,36 @@
+import './Calculator.css';
+import React from 'react';
+
+// eslint-disable-next-line react/prefer-stateless-function
```

Comment:
- [ ] Please do not disable Linters. Linters are tools that flag programming errors. Rather fix them.

Reply:
@FrederickMih The requirements of the project say that I have to use classes ... but eslint prefers pure functions that's why I disabled this part only ... Any suggestions?

Reply:
Hi @Peter1907 , you can try React pure component